unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/LoadingScreen/Scripts/LoadingScreenView.cs

```diff
@@ -0,0 +1,20 @@
+using UnityEngine;
+
+namespace DCL.LoadingScreen
+{
+    /// <summary>
+    /// Display the current state of the Loading Screen Controller
+    /// </summary>
+    public class LoadingScreenView : MonoBehaviour, ILoadingScreenView
+    {
+        private const string PATH = "_LoadingScreen";
+
+        public static LoadingScreenView Create() =>
```

Comment:
I would propose to inherit from `BaseComponentView` so it already has a method `Create`

Reply:
Done!

Reply:
Although, It raised some questions @mikhail-dcl 

1. By using the `Create` method from `BaseComponentView`, the path to the resource its no longer needed in `LoadinScreenView`. But then, Im now adding this path everytime I create the `LoadingScreenView`. Wouldnt this be error prone? Should I make a static variable in `LoadingScreenView` that points to the path?
2. Given that I'm instantiating as  `ILoadingScreenView` inside the controller, to be able to call the `Dispose()` pattern I have to re-define the method inside the interface. This is redundant, since this is already present as a virtual method in `BaseComponentView`.  WDYT of this redundancy? Its something acceptable? I see no really way around it, as I cannot declare `LoadingScreenView` in the controller as a `BaseComponentView` since I will have to cast it everytime I want to call any of its methods.

Reply:
1. I would do the following way in `LoadingScreenView`:
```
public static new LoadingScreenView Create() =>
            base.Create("<path>");
```

Reply:
2. Inheriting `ILoadingScreenView` from `IBaseComponentView` will solve the issue?